### PR TITLE
Add icons to project explorer tree for files with ".ts", ".jsx" and ".es6" extensions

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Add icons in project explorer tree for files with ".ts", ".jsx" and ".es6" extensions
ffff

### What issues does this PR fix or reference?
Issue: https://github.com/eclipse/che/issues/6089

### CODE REVIEW

**Comment by Ohrimenko1988** [comment]
ci-test

**Comment by riuvshin** [comment]
ci-test build report:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/732/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/732//Selenium_20tests_20report/)
[selenium tests report data](http://tests.codenvycorp.com/selenium_test_report_data/11043)
docker image: eclipseche/che-server:11043
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Comment by Ohrimenko1988** [comment]
ci-test

File: plugins/plugin-web/che-plugin-web-ext-web/src/main/resources/org/eclipse/che/plugin/web/client/jsx.svg [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    </defs>
+    <g clip-path="url(#_clipPath_QnU7mgg6ruLRsrEOIPBRSJZCiUmyZzg6)">
+        <g transform="matrix(1,0,0,1,4,3.744)">
+            <text transform="matrix(1,0,0,1,1.406,19.239)"
+                  style="font-family:'Open Sans';font-weight:400;font-size:18px;font-style:normal;fill:#4a86e8;stroke:none;">
+                JSX
+            </text>
+        </g>
+    </g>
+</svg>
```
Comment (vparfonov) [inline plugins/plugin-web/che-plugin-web-ext-web/src/main/resources/org/eclipse/che/plugin/web/client/jsx.svg]: add new line

**Review by vparfonov — APPROVED** [review]

**Comment by riuvshin** [comment]
ci-test build report:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/741/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/741//Selenium_20tests_20report/)
[selenium tests report data](http://tests.codenvycorp.com/selenium_test_report_data/11043)
docker image: eclipseche/che-server:11043
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Comment by Ohrimenko1988** [comment]
ci-test

**Comment by riuvshin** [comment]
ci-test build report:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/746/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/746//Selenium_20tests_20report/)
[selenium tests report data](http://tests.codenvycorp.com/selenium_test_report_data/11043)
docker image: eclipseche/che-server:11043
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Comment by Ohrimenko1988** [comment]
ci-test

**Comment by riuvshin** [comment]
ci-test build report:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/751/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/751//Selenium_20tests_20report/)
[selenium tests report data](http://tests.codenvycorp.com/selenium_test_report_data/11043)
docker image: eclipseche/che-server:11043
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Pull request merged**